### PR TITLE
Updating the packages to remove webmozart/path-util

### PR DESCRIPTION
**Pull request by mamazu**

### CODE REVIEW

**Comment by dantleech** [comment]
looks like conflict

File: composer.lock [hunk 1]
```diff
@@ -1571,16 +1571,16 @@
         },
         {
             "name": "phpactor/class-to-file",
-            "version": "dev-master",
+            "version": "0.5.1",
```
Comment (mamazu) [inline composer.lock:1574]: I don't know what happened here. That could be a downgrade.

Comment (dantleech) [reply composer.lock:1574]: yeah also not sure, but it's a downgrade indeed

**Review by dantleech — APPROVED** [review]

**Comment by dantleech** [comment]
tagged a new class-to-file and updated master, thanks!

**Comment by sitedyno** [comment]
Shouldn't this be changed now? https://github.com/phpactor/phpactor/blob/3ecebe2063f0efb875d5749d022745bd038f3652/lib/ConfigLoader/Adapter/PathCandidate/AbsolutePathCandidate.php#L7

**Comment by mamazu** [comment]
Well the aim of the merge request was to remove the packages depending on it. But you are right, this needs to be fixed.

**Comment by dantleech** [comment]
Oops, should really add that dep back or replace everything with the (urgh) "Filesystem" Path class.

**Comment by mamazu** [comment]
Should I add the replace in the `composer.json` or are we good now?

**Comment by dantleech** [comment]
the dependency is gone now or? nothing to do?

**Comment by mamazu** [comment]
Yep

**Pull request merged**